src/MoviePoster.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './css/MoviePoster.css'
+
+const MoviePoster = (props) => {
+  console.log('props>>>', props);
+  return (
+    <article className='movie-poster'>
+      <img src={props.posterPath} alt={props.title} />
+      <section className='movie-poster-details'>
+        <h2>{props.title}</h2>
```

Comment:
This is a great skeleton; I'll futz with the styling of the details beneath the poster.